crates/biome_html_parser/tests/html_specs/error/vue_unclosed_expression.vue.snap

```diff
@@ -0,0 +1,115 @@
+---
+source: crates/biome_html_parser/tests/spec_test.rs
+expression: snapshot
+---
+## Input
+
+```vue
+<template>
+	{{xyz
+</template>
+
+```
+
+
+## AST
+
+```
+HtmlRoot {
+    bom_token: missing (optional),
+    frontmatter: missing (optional),
+    directive: missing (optional),
+    html: HtmlElementList [
+        HtmlBogusElement {
+            items: [
+                HtmlOpeningElement {
+                    l_angle_token: L_ANGLE@0..1 "<" [] [],
+                    name: HtmlTagName {
+                        value_token: HTML_LITERAL@1..9 "template" [] [],
+                    },
+                    attributes: HtmlAttributeList [],
+                    r_angle_token: R_ANGLE@9..10 ">" [] [],
+                },
+                HtmlBogus {
+                    items: [
+                        L_DOUBLE_CURLY@10..14 "{{" [Newline("\n"), Whitespace("\t")] [],
+                        HTML_LITERAL@14..17 "xyz" [] [],
+                        HtmlContent {
+                            value_token: HTML_LITERAL@17..19 "<" [Newline("\n")] [],
+                        },
+                        HtmlContent {
+                            value_token: HTML_LITERAL@19..29 "/template>" [] [],
+                        },
+                    ],
+                },
+            ],
+        },
+    ],
+    eof_token: EOF@29..30 "" [Newline("\n")] [],
+}
+```
+
+## CST
+
+```
+0: HTML_ROOT@0..30
+  0: (empty)
+  1: (empty)
+  2: (empty)
+  3: HTML_ELEMENT_LIST@0..29
+    0: HTML_BOGUS_ELEMENT@0..29
+      0: HTML_OPENING_ELEMENT@0..10
+        0: L_ANGLE@0..1 "<" [] []
+        1: HTML_TAG_NAME@1..9
+          0: HTML_LITERAL@1..9 "template" [] []
+        2: HTML_ATTRIBUTE_LIST@9..9
+        3: R_ANGLE@9..10 ">" [] []
+      1: HTML_BOGUS@10..29
+        0: L_DOUBLE_CURLY@10..14 "{{" [Newline("\n"), Whitespace("\t")] []
+        1: HTML_LITERAL@14..17 "xyz" [] []
+        2: HTML_CONTENT@17..19
+          0: HTML_LITERAL@17..19 "<" [Newline("\n")] []
+        3: HTML_CONTENT@19..29
+          0: HTML_LITERAL@19..29 "/template>" [] []
+  4: EOF@29..30 "" [Newline("\n")] []
+
+```
+
+## Diagnostics
+
+```
+vue_unclosed_expression.vue:3:1 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × closing }} text expression
```

Comment:
This message could be better. Should explain that its "expecting" the thing in question.